src/bmpcodec.c

```diff
@@ -118,9 +118,8 @@ gdip_get_bmp_pixelformat (BITMAPINFOHEADER *bih, PixelFormat *dest)
 		case 1:
 			*dest = PixelFormat1bppIndexed;
 			break;
-	        default:
-			g_warning ("Unsupported bitcount (%d) and/or compression (%d).", bitCount, compression);
```

Comment:
I'd keep the warning message.

Reply:
This does end up causing messages to be printed to the log/console though. I can add back if you'd like but I'm not sure I see the value

Reply:
hmm I guess the value was in making sure all the possible combinations are covered. If we're sufficiently sure we covered everything then I'm fine with removing the warning.